src/applications/mhv/secure-messaging/tests/e2e/pages/PatientMessageDraftsPage.js

```diff
@@ -163,6 +163,14 @@ class PatientMessageDraftsPage {
     cy.get('[data-testid="delete-draft-button"]').click({ force: true });
   };
 
+  sendDraftMessage = draftMessage => {
+    cy.intercept('POST', '/my_health/v1/messaging/messages', draftMessage).as(
+      'sentDraftResponse',
+    );
+    cy.get('[data-testid="Send-Button"]').click({ force: true });
```

Comment:
Same as above would probably be applicable to locators - string values in constants file and pulled in here as a const variable.